PWGHF/TableProducer/candidateSelectorD0.cxx

```diff
@@ -174,9 +171,7 @@ struct HfCandidateSelectorD0 {
     if (candidate.decayLengthXY() > cuts->get(pTBin, "decay length XY")) {
       return false;
     }
-    if (candidate.decayLengthNormalised() * candidate.decayLengthNormalised() < 1.0) {
-      // return false; // add back when getter fixed
-    }
```

Comment:
I'm just wondering if this kind of cut (of course not with an hardcoded value) might be useful for non-prompt D analyses. @fgrosa @fcatalan92 what do you think?
Probably it's a minor point, since in line 165 there is the `minimum decay length` cut that does the same on the decay length, rather then the normalized one.

Reply:
@mfaggin these lines do not remove any cut, since it was commented (and not pT-differential). Apart from that, I would say that normalised decay length and normalised decay length XY should be similar (but also I would use them in a BDT for the non-prompt, I would not apply a linear cut at the level of the selector).

Reply:
thanks @fgrosa , it sounds good to me (and yes I saw that this was useless since the `return` was commented out, I was just thinking for a future fix and usage. But you already replied, thanks).

All good for me, let's wait the response of the others.